src/v/cluster/topic_updates_dispatcher.cc

```diff
@@ -54,12 +54,15 @@ topic_updates_dispatcher::apply_update(model::record_batch b) {
                   .then(
                     [this,
                      topic_assignments = std::move(topic_assignments),
-                     in_progress = std::move(in_progress)](std::error_code ec) {
+                     in_progress = std::move(in_progress),
+                     prio = del_cmd.value==model::kafka_consumer_offsets_nt]
```

Comment:
@mmaslankaprv, my guess for `delete_topic_cmd` is that it is `del_cmd.value` that holds the topic name to be deleted. Not sure what is there in `del_cmd.key` then?